src/components/Dropdown/__tests__/Dropdown-test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { generateVariantsFor } from '../../../utils/propsVariants';
+import { toPairs } from 'lodash';
+import Dropdown from '../Dropdown';
+import iconNames from '../../Icon/Icons';
+
+const variants = generateVariantsFor({
+  topText: [null, '', 'Resources'],
+  topIcon: [null, ...Object.keys(iconNames)],
+  items: [
+    [],
+    [
+      { value: 'Data Resource Protal', icon: 'file-size', onClick: () => {} },
+      { value: 'Website', icon: 'website', onClick: () => {} },
+      { value: 'Studies and Access', icon: 'study', onClick: () => {} },
+      { value: 'Support', icon: 'info', onClick: () => {} },
+      { value: 'Contact', icon: 'email', onClick: () => {} },
```

Comment:
be sure to add the new props to these for testing as well and change up `onClick` to